### DISCUSSION TITLE
What stats determines the Potato designation?

### DISCUSSION THREAD

**Discussion by Redbeanon** [razaqq/PotatoAlert#48] [General]
Love the program,

Still learning Python. What are the stats that designate a player as a potato? 

Regards
John

**Comment by razaqq** [comment]
Nobody gets designated as anything. It simply fetches the players statistics from the api

**Reply by Redbeanon** [reply]
I think I get it. the potato image after user name is who is using the program?

**Reply by razaqq** [reply]
Yes, nobody is being shamed for their stats
